libsolc/libsolc.cpp

```diff
@@ -229,12 +229,14 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 
 string compileMulti(string const& _input, bool _optimize, CStyleReadFileCallback _readCallback = nullptr)
 {
-	Json::Reader reader;
 	Json::Value input;
-	if (!reader.parse(_input, input, false))
+	std::stringstream inputStream(_input);
+	Json::CharReaderBuilder readerBuilder;
```

Comment:
Can you write a helper function that does all this for us? It could even re-use a static variable of type `Json::CharReaderBuilder`.

Reply:
Yeah sure!